test-fixtures/update.py

```diff
@@ -58,7 +58,7 @@ def default_stack():
     4. modify bitrise.yml (update stack value)
     '''
 
-    largest_semver = default_stack().split(pattern)[1]
+    default_semver = default_stack().split(pattern)[1]
```

Comment:
Would it be more useful for us to query the latest non-edge version of Bitrise stack instead?

Reply:
why?

Reply:
If we query the latest dot release, we can be notified when the dot release become available.

Reply:
😅 For now, we can just tell Github Actions not to open a PR to roll back the stack to an older version.